test/phonons.jl

```diff
@@ -0,0 +1,124 @@
+using Test
+using DFTK
+using LinearAlgebra
+using ForwardDiff
+using StaticArrays
+
+# ## Helper functions
```

Comment:
This comment doesn't add value. Something like "convert back and forth from array of Vec3 to columnwise matrix"